### PR TITLE
Add snail pet

### PR DESCRIPTION
**Pull request by WoofWoof0**
Hi there, I want to contribute to this repo by adding a new feature (pet snail). This is my first open-source PR, let me know if there is any problem, perhaps some guidance would be nice too. :)

### CODE REVIEW

File: src/panel/pets.ts [hunk 1]
```diff
@@ -191,6 +192,8 @@ export function createPet(
             return new Mod(...standardPetArguments, PetSpeed.normal);
         case PetType.totoro:
             return new Totoro(...standardPetArguments, PetSpeed.normal);
+        case PetType.snail:
+            return new Snail(...standardPetArguments,PetSpeed.slow);
```
Comment (tonybaloney) [inline src/panel/pets.ts:196]: I think `verySlow` would be a better speed

**Review by tonybaloney — CHANGES_REQUESTED** [review]
Please can you rename the GIFs so that they are all lower case. If you're using Windows you need to use the `git mv` command.

**Comment by tonybaloney** [comment]
This is a great PR, please can you look at my comments. Also please add an attribution for the artwork to README along with the others.

**Comment by WoofWoof0** [comment]
I had rename the gif files as requested.

**Review by tonybaloney — APPROVED** [review]

**Review by tonybaloney — APPROVED** [review]

**Pull request merged**